docs/home.md

```diff
@@ -1,4 +1,4 @@
-# Overview of TechPass
+# TechPass Overview
```

Comment:
Should this be "TechPass Overview" or "Overview" to match the sidebar -> a comment by Cyprian?

Reply:
done